portfolio.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>David Leiva</title>
+  </head>
+  <body>
+
+    <!-- Main navigation bar or toolbar -->
+    <header>
+      <label class="logo">My Logo</label>
+      <nav>
+        <input type="checkbox" id="menu">
+        <label for="menu" class="hamburger">
+          <div class="line"></div>
+          <div class="line"></div>
+          <div class="line"></div>
+        </label>
+        <ul class="links">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    
+    <!-- Headline Section -->
+    <section class="headline">
+      <h1>
+        Hey There.
+I’m Leiva.
+      </h1>
+      <h2>
+        I'm a Software Developer
+      </h2>
+      <p>
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hesitate to contact me.
+      </p>
+      <div class="social-links">
+        <a href="https://github.com/MasterLeiva" rel="noopener"><img src="img/github-icon.png" class="icon"/></a>
+        <a href="https://linkedin.com/in/1259221-d-leiva" rel="noopener"><img src="img/linkedin-icon.png" class="icon"/></a>
+        <a rel="noopener"><img src="img/angelist-icon.png" class="icon"/></a>
+        <a href="https://twitter.com/DavidLeiva111" rel="noopener"><img src="img/twitter-icon.png" class="icon"/></a>
+        <a rel="noopener"><img src="img/medium-icon.png" class="icon"/></a>
```

Comment:
- [ ] Dear @MasterLeiva, for social media It would be better to use an unordered list (`ul`) tag to wrap the content each being inside a `li` tag.  you can the put each `a` tags inside `li` tag. 
> something like 
```html 
<ul>
   <li><a rel="noopener"><img src="img/angelist-icon.png" class="icon"/></a></li>
   <li><a rel="noopener"><img src="img/angelist-icon.png" class="icon"/></a></li>
   <li><a rel="noopener"><img src="img/angelist-icon.png" class="icon"/></a></li>
</ul>
```